package.json

```diff
@@ -1,55 +1,28 @@
 {
-  "name": "uritemplate",
+  "name": "@apiaryio/uritemplate",
+  "version": "1.0.0",
   "description": "An UriTemplate implementation of rfc 6570",
-  "homepage": "https://www.github.com/fxa/uritemplate-js",
-  "keywords": [
-    "util",
-    "uri",
-    "template",
-    "rfc6570"
-  ],
+  "homepage": "https://github.com/apiaryio/uritemplate-js#readme",
   "author": "Franz X Antesberger",
-  "contributors": [],
-  "dependencies": {},
   "main": "bin/uritemplate.js",
   "scripts": {
-    "test": "jake clean build"
+    "test": "jake build"
   },
   "licenses": [{
       "type": "MIT",
       "url": "http://www.opensource.org/licenses/mit-license.php"
   }],
-  "files": [
-    "src",
-    "test",
-    "bin",
-    "build",
-    "uritemplate-test",
-    "Jakefile.js",
-    "own-testcases.json",
-    "demo.html",
-    "demo_AMD.html",
-    "README.md",
-    "require.js",
-    ".gitmodules",
-    "uritemplate-test/extended-tests.json",
-    "uritemplate-test/negative-tests.json",
-    "uritemplate-test/README.md",
-    "uritemplate-test/spec-examples-by-sections.json",
-    "uritemplate-test/spec-examples.json"
-  ],
-  "version": "0.3.4",
-  "readmeFilename": "README.md",
-  "gitHead": "901b85201a821427dfb4591b56aea3a70d45c67c",
   "devDependencies": {
       "jshint": "latest",
       "nodeunit": "latest",
       "jake": "latest",
-      "uglify-js": "latest",
       "async": "latest"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/fxa/uritemplate-js.git"
+    "url": "https://github.com/apiaryio/uritemplate-js.git"
+  },
+  "publishConfig": {
```

Comment:
It's a good practice against mistakes like cloning the repo and typing `npm publish` and accidentally publishing to the public registry or sth like that. We have that in multiple internal projects.